frontend/src/main/pages/AdminListAnnouncementsPage.js

```diff
@@ -1,15 +1,41 @@
-import React from 'react';
+import React from "react";
 import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+import AnnouncementTable from 'main/components/Announcement/AnnouncementTable';
+import { useBackend } from 'main/utils/useBackend';
+import { useCurrentUser } from "main/utils/currentUser";
+import { Button } from "react-bootstrap";
+import { useParams } from "react-router-dom";
 
-const AdminListAnnouncementsPage = () => {
-    // Stryker disable all : placeholder for future implementation
-  return (
-    <BasicLayout>
-      <div className="pt-2">
-        <h1>Admin List Announcements page not yet implemented</h1>
-      </div>
-    </BasicLayout>
-  )
-};
-
-export default AdminListAnnouncementsPage;
\ No newline at end of file
+export default function AdminListAnnouncementsPage()
+{
+    const { commonsId } = useParams();
+    const { data: currentUser} = useCurrentUser();
+
+    // Stryker disable all
+    const { data: announcements, error: _error, status: _status } = useBackend(
+        [`/api/announcements/all?commonsId=${commonsId}`],
+        { method: "GET", url: `/api/announcements/all`, params: { commonsId } },
```

Comment:
Check this against the URLs that are available on swagger.

I think you'll find that this URL doesn't exist.